### ISSUE TITLE
Per user persistent volume?

### ISSUE THREAD

**Issue by asapcatftw** [kasmtech/workspaces-issues#158]
To my understanding the current persistent volume is visible to all users who have access to the image.  Is it possible to dynamically mount volumes to images on runtime based on user_id?  This way each user has a personal persistent drive.

For example
```
{
   "/var/kasm_user_share/{user_id}":{
      "bind":"/share",
      "mode":"rw",
      "uid": 1000,
      "gid": 1000,
      "required": true
   }
}
```

**Comment by asapcatftw** [comment]
I am currently using the persistent profile option but updating images causes the shortcuts to break. I think the per-user persistent volume would be a good feature.

**Comment by j-travis** [comment]
Hi @asapcatftw , I assumed since you closed the ticket you found the answer to your question.

You indeed can create your own per-user persistent share (separate from the persistent profile feature) , buy utilizing the `{user_id}` or `{username}` tokens in the mount path just like you described above.

We will update the docs  for the volume mapping features as its not covered there

**Comment by asapcatftw** [comment]
Sorry for the confusion, I didn't test the per-user persistent share. I don't know if it's possible to use `{user_id}` in the field.
I closed the issue because I started using a persistent profile.

Also, the docs have Persistent Data and volume mapping config in the How-To guide duplicated. 
![image](https://user-images.githubusercontent.com/31966895/155350005-a5342544-3335-447c-b394-b9363cf5ec94.png)
